selfdrive/car/honda/interface.py

```diff
@@ -5,7 +5,7 @@
 from common.realtime import DT_CTRL
 from selfdrive.swaglog import cloudlog
 from selfdrive.config import Conversions as CV
-from selfdrive.controls.lib.drive_helpers import create_event, EventTypes as ET, get_events
+from selfdrive.controls.lib.events import ET
```

Comment:
If we are importing this I'm thinking there's a way this can be refactored. Why does only Honda import this?

Reply:
The code that uses this is pretty old and can probably just be deleted, but I don't want to make this PR bigger and harder to test. Will fix after merging